how-to/networking/timedatectl-and-timesyncd.md

```diff
@@ -1,7 +1,11 @@
 (timedatectl-and-timesyncd)=
 # Synchronise time using timedatectl and timesyncd
 
-Ubuntu uses `timedatectl` and `timesyncd` for synchronising time, and they are installed by default as part of `systemd`. You can optionally use `chrony` to {ref}`serve the Network Time Protocol <serve-ntp-with-chrony>`.
+Ubuntu can use `timedatectl` and `timesyncd` for synchronising time, which can be installed as follows, replacing the default `chrony` installation. You can optionally use `chrony` as a {ref}`Network Time Security (NTS) client <chrony-client>` or to {ref}`serve the Network Time Protocol <serve-ntp-with-chrony>`.
```

Comment:
please-add:
I think in this place where we say "this is how you change from chrony to timesyncd" it deserves yet another "since 25.10 formerly it was ...". I assume for a while the majority of readers will be 24.04 users and that helps them in case they did not come by the other pages that already state this.